services/ui-src/package.json

```diff
@@ -12,6 +12,7 @@
     "bootstrap": "^5.1.3",
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.6",
+    "node": "^16.13.2",
```

Comment:
What's going on here? I think we're limited to node 14, and also why are we installing all of node as a package?

Reply:
I kept getting `eslint@8.14.0: The engine "node" is incompatible with this module. Expected version "^12.22.0 || ^14.17.0 || >=16.0.0". Got "14.16.0"` is that just a me problem?

Reply:
When I remove that from my local I'm not getting the same issue